test/serverStatus.test.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import { ServerStatusView } from '../src/languageservice/serverStatus';
+import { assert } from 'chai';
+import Constants = require('../src/constants/constants');
+
+suite('Server Status View Tests', () => {
+
+    let serverStatusView: ServerStatusView;
+
+    setup(() => {
+        serverStatusView = new ServerStatusView();
+    });
+
+    test('Test installing service status', () => {
+        serverStatusView.installingService();
+        setTimeout(() => {
```

Comment:
This test is always going to pass. It'll call setTimeout but that returns immediately at which point the test will finish and pass. (you can test this by adding a throws in the callback). You can either wrap the setTimeout in a Promise and then resolving/rejecting within the setTimeout callback based on that or pass in done and then call done() at the end of the callback.